threads.php

```diff
@@ -375,3 +380,58 @@ function cfth_timeline_css() {
 <?php
 }
 
+// Meta field markup for add thread page
+function threads_taxonomy_add_new_meta_field() {
```

Comment:
Pay attention to your function prefixes. This doesn't match the prefix used by all legacy functions.